proto/viam/app/v1/billing.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+package viam.app.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/v1";
+
+service BillingService {
+  rpc GetCurrentMonthUsageSummary(GetCurrentMonthUsageSummaryRequest) returns (GetCurrentMonthUsageSummaryResponse);
+  rpc GetUnpaidBalance(GetUnpaidBalanceRequest) returns (GetUnpaidBalanceResponse);
+  rpc GetInvoiceHistory(GetInvoiceHistoryRequest) returns (GetInvoiceHistoryResponse);
+  rpc GetItemizedInvoice(GetItemizedInvoiceRequest) returns (GetItemizedInvoiceResponse);
+  rpc GetBillingSummary(GetBillingSummaryRequest) returns (GetBillingSummaryResponse);
+}
+
+message GetCurrentMonthUsageSummaryRequest {
+  string org_id = 1;
+}
+
+message CurrentMonthUsageSummary {
+  double cloud_storage_usage = 1;
+  double data_upload_usage = 2;
+  double data_egres_usage = 3;
+  double standard_compute_usage = 4;
+  double total_usage_with_discount = 5; // returns number with any discounts applied
+  double total_usage_without_discount = 6; // returns number without any discounts applied
```

Comment:
should the discount be calculated separately? I feel like the name implies usage, billing independent

Reply:
For the FE, we want to show the total usage with the discount and without. The actual numbers we show here are usage in terms of $ cost, not like GB usage etc. The name here can change but its referencing all the usage stats for the current month. The reason we shouldn't separate this out is because we calculate this on read i.e. these statistics are not computed every time there is an update. 

So if we wanted to separate this into with and without discounts, we would then be making 2 requests that would duplicate calls to the aggregator functions. One that applies a discount after and one that does not. This will strain the API for no reason, so doesn't make sense to me to separate it out.

Reply:
ah okay, I didn't realize that the usage was in terms of $ cost. will there be separate protos for usage in terms of GB usage/etc? if so, can we differentiate naming?

Reply:
Currently not needed in the designs, but i can change this to make it more clear its $ cost in case we want to add GB usage late